frontend/bank-transfer-ui/src/styles.css

```diff
@@ -10,9 +10,49 @@
   font-style: normal;
 }
 
-body {
-  min-height: 100vh !important;
+html, body {
+  height: 100% !important;
```

Comment:
Jeśli użyłbyś fixed zamiast absolute dla footera, to zakładam, że to mogłoby zniknąć

Reply:
Niestety footer nie może być fixed, gdyż strona może zwiększyć swoja wysokość z zależności od ilości contentu.

Reply:
ok